Description: use Debian paths instead of silly ones

This doesn't need to be sent upstream
--- smokeping-2.6.5.orig/etc/config.dist.in
+++ smokeping-2.6.5/etc/config.dist.in
@@ -7,13 +7,13 @@ sendmail = @SENDMAIL@
 # NOTE: do not put the Image Cache below cgi-bin
 # since all files under cgi-bin will be executed ... this is not
 # good for images.
-imgcache = @prefix@/cache
+imgcache = /var/cache/smokeping
 imgurl   = cache
-datadir  = @prefix@/data
-piddir  = @prefix@/var
+datadir  = /var/lib/smokeping
+piddir  = /var/run/smokeping
 cgiurl   = http://some.url/smokeping.cgi
-smokemail = @prefix@/etc/smokemail.dist
-tmail = @prefix@/etc/tmail.dist
+smokemail = /etc/smokeping/smokemail
+tmail = /etc/smokeping/tmail
 # specify this to get syslog logging
 syslogfacility = local0
 # each probe is now run in its own process
@@ -47,7 +47,7 @@ AVERAGE  0.5 144   720
 
 *** Presentation ***
 
-template = @prefix@/etc/basepage.html.dist
+template = /etc/smokeping/basepage.html
 
 + charts
 
diff --git a/etc/config.dist.in b/etc/config.dist.in
index 6d7bce9..8a6c8be 100644
--- a/etc/config.dist.in
+++ b/etc/config.dist.in
@@ -105,10 +105,10 @@ unison_tolerance = 2
 
 + FPing
 
-binary = /usr/sbin/fping
+binary = /usr/bin/fping
 
 *** Slaves ***
-secrets=@prefix@/etc/smokeping_secrets.dist
+secrets=/etc/smokeping/smokeping_secrets
 #+boomer
 #display_name=boomer
 #color=0000ff
